src/SpotifyApi.NetCore/IFollowApi.cs

```diff
@@ -93,5 +93,119 @@ public interface IFollowApi
+        Task FollowPlaylist(
+            string playlistId,
+            bool isPublic = true,
+            string accessToken = null
+            );
+        #endregion
+
+        #region GetUsersFollowedArtists
+        /// <summary>
+        /// Get User's Followed Artists
+        /// </summary>
+        /// <param name="limit">Optional. The maximum number of items to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="after">Optional. The last artist ID retrieved from the previous request.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedArtists"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/get-followed/
+        /// </remarks>
+        Task<PagedArtists> GetUsersFollowedArtists(
+            int limit = 20,
+            string after = null,
+            string accessToken = null
+            );
+
+        /// <summary>
+        /// Get User's Followed Artists
+        /// </summary>
+        /// <param name="limit">Optional. The maximum number of items to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="after">Optional. The last artist ID retrieved from the previous request.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedArtists"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/get-followed/
+        /// </remarks>
+        Task<T> GetUsersFollowedArtists<T>(
+            int limit = 20,
+            string after = null,
+            string accessToken = null
+            );
+        #endregion
+
+        #region UnfollowArtistsOrUsers
+        /// <summary>
+        /// Unfollow Artists or Users
+        /// </summary>
+        /// <param name="artistIds">A comma-separated list of the artist Spotify IDs. A maximum of 50 artist IDs can be sent in one request. A minimum of 1 artist id is required.</param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/unfollow-artists-users/
+        /// </remarks>
+        Task UnfollowArtists(
+            string[] artistIds,
+            string accessToken = null
+            );
+
+        /// <summary>
+        /// Unfollow Artists or Users
+        /// </summary>
+        /// <param name="userIds">A comma-separated list of the user Spotify IDs. A maximum of 50 user IDs can be sent in one request. A minimum of 1 user id is required. </param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/unfollow-artists-users/
+        /// </remarks>
+        Task UnfollowUsers(
+            string[] userIds = null,
+            string accessToken = null
+            );
+        #endregion
+
+        #region UnfollowPlaylist
+        /// <summary>
+        /// Unfollow a Playlist
```

Comment:
This should be "Remove the current user as a follower of a playlist." as per docs

Reply:
Hi Daniel,

Here you are not going by the title in the docs you are going by the full description.  I am going by the title everywhere.  Need to get a decision on the title or full sentence description for this and I change it once everywhere.

Thanks for your patience and time.

Best Regards,

Akshay Srinivasan

Reply:
That's right. The title is used for the Method name, i.e. "Follow a Playlist" = `FollowPlaylist`

The description is used for the `<summary>` tag, i.e.

![image](https://user-images.githubusercontent.com/2294760/85985240-8b896f00-ba3e-11ea-96ba-c263e9dc0502.png)

Reply:
Hehe did not need the image though *giggles* but I am happy now I was very depressed that I could not pass this code. ♥♥♥